### PR TITLE
Fix minor whoopsies in clonerefs/initupload

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Fix SIGSEGV in clonerefs error handling logic

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

Configure user.{name,email} in clonerefs image

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

Validate GCS options in initupload correctly

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

/cc @cjwagner @BenTheElder

### CODE REVIEW

File: prow/cmd/initupload/main.go [hunk 1]
```diff
@@ -66,11 +66,11 @@ func main() {
 	flag.Parse()
 
 	if !*dryRun {
-		if *gcsBucket != "" {
+		if *gcsBucket == "" {
```
Comment (stevekuznetsov) [inline prow/cmd/initupload/main.go]: Will add options `struct` & tests in a followup

File: prow/cmd/clonerefs/Dockerfile [hunk 2]
```diff
@@ -15,5 +15,6 @@
 FROM gcr.io/k8s-prow/git:0.1
 LABEL maintainer="skuznets@redhat.com"
 
+RUN git config --global user.name ci-robot && git config --global user.email ci-robot@k8s.io
```
Comment (BenTheElder) [inline prow/cmd/clonerefs/Dockerfile]: can we do this at runtime instead? it seems like this should be configurable for different prow deployments without building a new image

Comment (stevekuznetsov) [reply prow/cmd/clonerefs/Dockerfile]: Sure, I can add flags on `clonerefs` to expose these two instead

**Comment by stevekuznetsov** [comment]
@bentheelder updated

**Pull request merged**